express-get/server.ts

```diff
@@ -0,0 +1,58 @@
+import express from 'express';
+import pg from 'pg';
+import { errorMiddleware } from './lib/index.js';
+import { ClientError } from './lib/client-error.js';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/pagila',
+  ssl: {
+    // Allow non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.use(errorMiddleware);
```

Comment:
this should be below the routes also!

Error middleware should always be at the end of the file